hierarchicalforecast/core.py

```diff
@@ -76,11 +76,13 @@ def reconcile(self,
         `S`: pd.DataFrame with summing matrix of size `(base, bottom)`, see [aggregate method](https://nixtla.github.io/hierarchicalforecast/utils.html#aggregate).<br>
         `tags`: Each key is a level and its value contains tags associated to that level.<br>
         `level`: float list 0-100, confidence levels for prediction intervals.<br>
-        `bootstrap`: bool, whether or not to use bootstraped prediction intervals, alternative normality assumption.<br>
-
+        `intervals_method`: str, method used to calculate prediction intervals, one of `normality`, `bootstrap`, `permbu`.<br>
+         
         **Returns:**<br>
         `y_tilde`: pd.DataFrame, with reconciled predictions.        
         """
+        if intervals_method not in ['normality', 'bootstrap', 'permbu']:
```

Comment:
Is the 'normality' method refering to 'mint_normality'?
I think the name could be a source of confusion.

Reply:
hey @kdgutier! I've changed the name to `normality` since it is a general method that works independently of the reconciliation method: https://otexts.com/fpp3/rec-prob.html

Reply:
Cool, would it be possible to explain it in the docstring with the pointer to the link?